libraries/AP_InertialSensor/AP_InertialSensor_BMI270.cpp

```diff
@@ -0,0 +1,573 @@
+    BMI270_REG_SENSORTIME_2 = 0x1A,
+    BMI270_REG_EVENT = 0x1B,
+    BMI270_REG_INT_STATUS_0 = 0x1C,
+    BMI270_REG_INT_STATUS_1 = 0x1D,
+    BMI270_REG_INTERNAL_STATUS = 0x21,
+    BMI270_REG_TEMPERATURE_LSB = 0x22,
+    BMI270_REG_TEMPERATURE_MSB = 0x23,
+    BMI270_REG_FIFO_LENGTH_LSB = 0x24,
+    BMI270_REG_FIFO_LENGTH_MSB = 0x25,
+    BMI270_REG_FIFO_DATA = 0x26,
+    BMI270_REG_ACC_CONF = 0x40,
+    BMI270_REG_ACC_RANGE = 0x41,
+    BMI270_REG_GYRO_CONF = 0x42,
+    BMI270_REG_GYRO_RANGE = 0x43,
+    BMI270_REG_AUX_CONF = 0x44,
+    BMI270_REG_FIFO_DOWNS = 0x45,
+    BMI270_REG_FIFO_WTM_0 = 0x46,
+    BMI270_REG_FIFO_WTM_1 = 0x47,
+    BMI270_REG_FIFO_CONFIG_0 = 0x48,
+    BMI270_REG_FIFO_CONFIG_1 = 0x49,
+    BMI270_REG_SATURATION = 0x4A,
+    BMI270_REG_INT1_IO_CTRL = 0x53,
+    BMI270_REG_INT2_IO_CTRL = 0x54,
+    BMI270_REG_INT_LATCH = 0x55,
+    BMI270_REG_INT1_MAP_FEAT = 0x56,
+    BMI270_REG_INT2_MAP_FEAT = 0x57,
+    BMI270_REG_INT_MAP_DATA = 0x58,
+    BMI270_REG_INIT_CTRL = 0x59,
+    BMI270_REG_INIT_DATA = 0x5E,
+    BMI270_REG_ACC_SELF_TEST = 0x6D,
+    BMI270_REG_GYR_SELF_TEST_AXES = 0x6E,
+    BMI270_REG_PWR_CONF = 0x7C,
+    BMI270_REG_PWR_CTRL = 0x7D,
+    BMI270_REG_CMD = 0x7E,
+};
+
+/**
+* The following device config microcode has the following copyright:
+*
+* Copyright (c) 2020 Bosch Sensortec GmbH. All rights reserved.
+*
+* BSD-3-Clause
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions are met:
+*
+* 1. Redistributions of source code must retain the above copyright
+*    notice, this list of conditions and the following disclaimer.
+*
+* 2. Redistributions in binary form must reproduce the above copyright
+*    notice, this list of conditions and the following disclaimer in the
+*    documentation and/or other materials provided with the distribution.
+*
+* 3. Neither the name of the copyright holder nor the names of its
+*    contributors may be used to endorse or promote products derived from
+*    this software without specific prior written permission.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+* "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+* LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+* FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+* COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+* INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+* (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+* SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+* HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+* STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+* IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*/
+static const uint8_t bmi270_maximum_fifo_config_file[] = { BMI270_REG_INIT_DATA,
```

Comment:
this should be a static const class member

Reply:
Fixed